From 79ccf2e451cfc6c4aed300a11bf105bd9b26b57a Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 10:29:53 +0100
Subject: [PATCH 28/53] func_tests: drop more af_family occurences
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Again, af_family is set but not used. Instead pf_class is the one
in charge. Fixes:

  test_fragments.c:89:16: warning: variable ‘af_family’ set but not used [-Wunused-but-set-variable]
  test_inaddr_any.c:82:16: warning: variable ‘af_family’ set but not used [-Wunused-but-set-variable]
  test_sockopt.c:81:16: warning: variable ‘af_family’ set but not used [-Wunused-but-set-variable]
  test_timetolive.c:104:16: warning: variable ‘af_family’ set but not used [-Wunused-but-set-variable]
  test_sctp_sendrecvmsg.c:80:16: warning: variable ‘af_family’ set but not used [-Wunused-but-set-variable]

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/func_tests/test_fragments.c        | 4 +---
 src/func_tests/test_inaddr_any.c       | 4 +---
 src/func_tests/test_sctp_sendrecvmsg.c | 4 +---
 src/func_tests/test_sockopt.c          | 4 +---
 src/func_tests/test_timetolive.c       | 4 +---
 5 files changed, 5 insertions(+), 15 deletions(-)

diff --git a/src/func_tests/test_fragments.c b/src/func_tests/test_fragments.c
index 973d485..27c1c4a 100644
--- a/src/func_tests/test_fragments.c
+++ b/src/func_tests/test_fragments.c
@@ -86,7 +86,7 @@ main(int argc, char *argv[])
 	struct sctp_sndrcvinfo *sinfo;
         struct iovec out_iov;
         int error, bytes_sent;
-	int pf_class, af_family;
+	int pf_class;
 	uint32_t ppid;
 	uint32_t stream;
 	sctp_assoc_t associd1, associd2;
@@ -105,7 +105,6 @@ main(int argc, char *argv[])
 	/* Set some basic values which depend on the address family. */
 #if TEST_V6
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
         loop1.v6.sin6_family = AF_INET6;
         loop1.v6.sin6_addr = in6addr_loopback;
@@ -116,7 +115,6 @@ main(int argc, char *argv[])
         loop2.v6.sin6_port = htons(SCTP_TESTPORT_2);
 #else
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
         loop1.v4.sin_family = AF_INET;
         loop1.v4.sin_addr.s_addr = SCTP_IP_LOOPBACK;
diff --git a/src/func_tests/test_inaddr_any.c b/src/func_tests/test_inaddr_any.c
index 29e8e6d..722a702 100644
--- a/src/func_tests/test_inaddr_any.c
+++ b/src/func_tests/test_inaddr_any.c
@@ -79,7 +79,7 @@ main(void)
         char *telephone = "Watson, come here!  I need you!\n";
         char *telephone_resp = "I already brought your coffee...\n";
         int error;
-	int pf_class, af_family;
+	int pf_class;
 	uint32_t ppid;
 	uint32_t stream;
 	socklen_t namelen;
@@ -92,7 +92,6 @@ main(void)
 	/* Set some basic values which depend on the address family. */
 #if TEST_V6
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
         loop.v6.sin6_family = AF_INET6;
         loop.v6.sin6_addr = (struct in6_addr)SCTP_IN6ADDR_LOOPBACK_INIT;
@@ -103,7 +102,6 @@ main(void)
         anyaddr.v6.sin6_port = 0;
 #else
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
         loop.v4.sin_family = AF_INET;
         loop.v4.sin_addr.s_addr = SCTP_IP_LOOPBACK;
diff --git a/src/func_tests/test_sctp_sendrecvmsg.c b/src/func_tests/test_sctp_sendrecvmsg.c
index de85820..4e3e9b7 100644
--- a/src/func_tests/test_sctp_sendrecvmsg.c
+++ b/src/func_tests/test_sctp_sendrecvmsg.c
@@ -77,7 +77,7 @@ int main(int argc, char *argv[])
 	sockaddr_storage_t loop2;
 	sockaddr_storage_t msgname;
 	int error;
-	int pf_class, af_family;
+	int pf_class;
 	uint32_t ppid;
 	uint32_t stream;
 	struct sctp_event_subscribe subscribe;
@@ -100,7 +100,6 @@ int main(int argc, char *argv[])
 	/* Set some basic values which depend on the address family. */
 #if TEST_V6
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
 	loop1.v6.sin6_family = AF_INET6;
 	loop1.v6.sin6_addr = in6addr_loopback;
@@ -111,7 +110,6 @@ int main(int argc, char *argv[])
 	loop2.v6.sin6_port = htons(SCTP_TESTPORT_2);
 #else
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
 	loop1.v4.sin_family = AF_INET;
 	loop1.v4.sin_addr.s_addr = SCTP_IP_LOOPBACK;
diff --git a/src/func_tests/test_sockopt.c b/src/func_tests/test_sockopt.c
index 5c7607d..a31c5d3 100644
--- a/src/func_tests/test_sockopt.c
+++ b/src/func_tests/test_sockopt.c
@@ -78,7 +78,7 @@ main(void)
         struct iovec out_iov;
         char *message = "hello, world!\n";
         int error;
-	int pf_class, af_family;
+	int pf_class;
 	uint32_t ppid;
 	uint32_t stream;
 	sctp_assoc_t udp_svr_associd, udp_clt_associd;
@@ -109,7 +109,6 @@ main(void)
 	/* Set some basic values which depend on the address family. */
 #if TEST_V6
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
         udp_svr_loop.v6.sin6_family = AF_INET6;
         udp_svr_loop.v6.sin6_addr = in6addr_loopback;
@@ -128,7 +127,6 @@ main(void)
         tcp_clt_loop.v6.sin6_port = htons(SCTP_TESTPORT_1+3);
 #else
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
         udp_svr_loop.v4.sin_family = AF_INET;
         udp_svr_loop.v4.sin_addr.s_addr = SCTP_IP_LOOPBACK;
diff --git a/src/func_tests/test_timetolive.c b/src/func_tests/test_timetolive.c
index 620ad3e..9e41cf5 100644
--- a/src/func_tests/test_timetolive.c
+++ b/src/func_tests/test_timetolive.c
@@ -101,7 +101,7 @@ int main(int argc, char *argv[])
 	struct sctp_sndrcvinfo *sinfo;
         struct iovec out_iov;
         int error;
-	int pf_class, af_family;
+	int pf_class;
 	uint32_t ppid;
 	uint32_t stream;
 	sctp_assoc_t associd1, associd2;
@@ -122,7 +122,6 @@ int main(int argc, char *argv[])
 	/* Set some basic values which depend on the address family. */
 #if TEST_V6
 	pf_class = PF_INET6;
-	af_family = AF_INET6;
 
         loop1.v6.sin6_family = AF_INET6;
         loop1.v6.sin6_addr = in6addr_loopback;
@@ -133,7 +132,6 @@ int main(int argc, char *argv[])
         loop2.v6.sin6_port = htons(SCTP_TESTPORT_2);
 #else
 	pf_class = PF_INET;
-	af_family = AF_INET;
 
         loop1.v4.sin_family = AF_INET;
         loop1.v4.sin_addr.s_addr = SCTP_IP_LOOPBACK;
-- 
1.7.11.7

